primitives/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "ac-primitives"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+codec = { package = 'parity-scale-codec', version = "2.0.0", default-features = false,  features = ['derive']}
+
+sp-core = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+
+[features]
+default = ["std"]
+std = ["codec/std"]
```

Comment:
Could you add sp-core/std as well? Sorry to be picky here, but I've run into problems already due to missing std imports.

Reply:
this is absolutely not picky, but a necessity! I know that these things can cause unforeseen problems!